src/main/java/lotto/Exception/BudgetCheck.java

```diff
@@ -0,0 +1,19 @@
+package lotto.Exception;
+
+public class BudgetCheck {
+    //예외처리
+    private static final int LOTTO_ONE_PRICE = 1000;
+
+    public int check(int budget) {
+        if ((budget % LOTTO_ONE_PRICE) == 0) {
+            System.out.println();
+            return budget / LOTTO_ONE_PRICE;
+        }
+
+        try {
+            throw new IllegalAccessException("[ERROR] 1000원으로 나뉘어 떨어져야합니다.");
+        } catch (IllegalAccessException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
들어온 budget이 잘못된 상황이라 IllegalArgumentsException이 적합해보여요!
throw는 던지는 곳에서는 굳이 잡아줄 필요 없을 것 같아요! 이미 RuntimeException 들을 상속하고 있는 친구들이라 굳이 변환을 해줄 필요도 없구요!
```suggestion
        throw new IllegalArgumentsException("[ERROR] 1000원으로 나뉘어 떨어져야합니다.");
```